config/config.go

```diff
@@ -185,6 +185,7 @@ type ComponentStatuses struct {
 type ComponentStatus struct {
 	AppLabel  string `yaml:"app_label,omitempty"`
 	IsCore    bool   `yaml:"is_core,omitempty"`
+	IsProxy   bool   `yaml:"is_proxy,omitempty"`
```

Comment:
Note that his change requires a operator change at least in the def and defaults
https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L508

Reply:
oh wow, good catch.